### DISCUSSION TITLE
The default HeartMaxFallSpeed

### DISCUSSION THREAD

**Discussion by Nightingale08** [Jcw87/c2-sans-fight#121] [Q&A] [answered]
Hi, whats the default HeartMaxFallSpeed?

**Comment by thelift112** [comment]
I don't know but self I'm currently trying to figure out the movement speed cause I want a platform to be moving at the same speed as the soul. I think the movement speed is 150, and @Nightingale08 you can take an figure out the speed by having like a bone fly in the same direction as soul when SansSlam is called and change until they move both at the same speed.

**Reply by Nightingale08** [reply]
Ok thanks

**Reply by thelift112** [reply]
> Ok thanks

Hey did you find out what it is?

**Reply by Nightingale08** [reply]
not yet, im planning to move to "Create your Frisk" to make actual battles and maybe an overworld, (if i can get a pixel artist) but im currently on my break for now.

**Reply by thelift112** [reply]
> not yet, im planning to move to "Create your Frisk" to make actual battles and maybe an overworld, (if i can get a pixel artist) but im currently on my break for now.

I wish I could move to Create Your Frisk but I think it will be too hard to learn... pretty sure there are like no tutorials on how to make an fight and that...

**Reply by Nightingale08** [reply]
ok

**Comment by Jcw87** [comment, marked as answer]
The default MaxFallSpeed is 750. There is a `ResetVars` function that is used internally at the start and end of every attack. It sets several variables (including MaxFallSpeed) back to their defaults.

**Reply by Nightingale08** [reply]
Thankss, ill maybe do a small update on my sans battles.
